src/index.js

```diff
@@ -128,6 +128,11 @@ const highway = {
 
     return true
   },
+  
+  // return the current route
+  cr () {
```

Comment:
@azerothl We should name it more explicitly. e.g. `currentRoute` or just the same name as Backbone `getFragment`.

Reply:
currentRoute is good for me, I think it's more understandable than getFragment

Reply:
:+1: Just push the modification to your fork and I'll merge it and release a new version